tests/conftest.py

```diff
@@ -1623,6 +1623,106 @@ def list_callback(arg=None):
     )
 
 
+@pytest.fixture
```

Comment:
As your test suite grows, you should start considering scopes for your fixtures, to avoid repeating setup steps when not needed: https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Reply:
@dantonpimentel thanks for the link, I've created a separate task to tackle this in the future -- we need to improve our maintainability of the test suites so this is very useful.